backend/src/main/java/com/bakdata/conquery/models/types/ResultType.java

```diff
@@ -233,9 +242,103 @@ public List<List<Integer>> getFromResultSetAsList(ResultSet resultSet, int colum
 		}
 	}
 
-	@CPSType(id = "STRING", base = ResultType.class)
+	@Data
+	public static class MappedMultiStringT extends ResultType<List<String>> {
```

Comment:
Ich denke mit diesen Klassen wird das ResultType System weniger definiert und wird sollten überlegen, wie man dass besser refactored, so dass die Logik für das erstellen des Ergebisses aus dem ResultType heraus wandert und näher am Select und seinem Mapping drannen ist.

Mir fällt das erst jetzt auf, dass wir durch abschaffen des CPSTypes in der Klasse, auch implizite Konventionen abgeschafft haben.
```
public String typeInfo() {
			return "LIST[STRING]";
		}
```